### PR TITLE
Fix Friendica node info json

### PR DESCRIPTION
**Pull request by fabrixxm**
And stop worker sending warning from PortableContacts.php

### CODE REVIEW

File: mod/friendica.php [hunk 1]
```diff
@@ -52,7 +52,7 @@ function friendica_init(App $a)
 			'locked_features'  => $locked_features,
 			'explicit_content' => (int)Config::get('system', 'explicit_content', false),
 			'language'         => Config::get('system','language'),
-			'register_policy ' => $register_policy[intval(Config::get('config', 'register_policy'))],
+			'register_policy' => $register_policy[intval(Config::get('config', 'register_policy'))],
```
Comment (MrPetovan) [inline mod/friendica.php]: Please add a space to match the arrows' alignment.

File: src/Protocol/PortableContact.php [hunk 2]
```diff
@@ -1375,7 +1375,12 @@ public static function checkServer($server_url, $network = "", $force = false)
 					}
 
 					$info = $data['info'];
-					$register_policy = constant($data['register_policy']);
+					if (in_array($data['register_policy'], ['REGISTER_CLOSED', 'REGISTER_APPROVE', 'REGISTER_OPEN'])) {
+						$register_policy = constant($data['register_policy']);
+					} else {
+						Logger::log("Register policy '".$data['register_policy']."' from $server_url is invalid.");
```
Comment (MrPetovan) [inline src/Protocol/PortableContact.php]: If you're going for double-quotes, you can write `"Register policy '{$data['register_policy']}' from $server_url is invalid."`.

Either that or wrap operators with spaces.

**Comment by MrPetovan** [comment]
Fixes #6087

**Comment by annando** [comment]
Ooops ...

**Comment by fabrixxm** [comment]
You could squash this PR before merge, when ok

**Comment by MrPetovan** [comment]
Follow-up to #6080

In hindsight I don't think the check was necessary, the notice appeared on stable site because develop sites weren't including the expected `'register_policy'` key. So this likely won't fail anymore again.

**Comment by fabrixxm** [comment]
The check is necessary to stop my server to send me warning every 10 minutes until everyone update :-P

**Comment by MrPetovan** [comment]
It doesn't give you `Notice: Undefined index: register_policy in src/Protocol/PortableContact.php:1377` notices?

**Comment by fabrixxm** [comment]
Oh.. ouf..  :-(
no, I think my prod server has notices off.. It should. It should need another check...

**Pull request merged**